src/app/pages/Designathons/Designathon24/Designathon24.module.scss

```diff
@@ -0,0 +1,174 @@
+:root {
+	--pink: #ff75a4;
+	--hot-pink: #f84e72;
+	--peach: #ff6969;
+	--cream: #fff5e0;
+	--black: #242424;
+}
```

Comment:
this caused a regression, if you're adding new css variables into the global scope, please verify no other parts of the site are broken